contracts/lib/LibPrices.sol

```diff
@@ -79,7 +79,7 @@ library Prices {
                 /* handle underflow */
                 return 0;
             } else {
-                return globalLeverage - uint256(delta);
+                return globalLeverage - (oldLeverage - newLeverage);
```

Comment:
Why do this? Also this changes the logic right, since `delta = newLeverage - oldLeverage`, so previously this was `globalLeverage - (newLeverage - oldLeverage)`